source/portable/os/ota_os_posix.c

```diff
@@ -117,27 +122,49 @@ OtaOsStatus_t Posix_OtaSendEvent( OtaEventContext_t * pEventCtx,
                                   unsigned int timeout )
```

Comment:
mq_open should be called with the O_NONBLOCK flag.

Reply:
Fixed.